hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -245,6 +250,16 @@ public int getMaxConsistencyCheckIntervalMs() {
     return Integer.parseInt(props.getProperty(MAX_CONSISTENCY_CHECK_INTERVAL_MS_PROP));
   }
 
+  public BulkInsertSortMode getBulkInsertSortMode() {
+    String sortMode = props.getProperty(BULKINSERT_SORT_MODE);
+    try {
+      return BulkInsertSortMode.valueOf(sortMode.toUpperCase());
+    } catch (IllegalArgumentException e) {
```

Comment:
given IllegalArgumentException is itself runtime exception.. may be ok to just let that percolate.